external-crates/move/tooling/prettier-move/README.md

```diff
@@ -0,0 +1,88 @@
+# Prettier Move Plugin
+
+This is a Move language plugin for the
+[Prettier](https://prettier.io/) code formatter. It uses a Move
+[parser](https://github.com/tzakian/tree-sitter-move) built on top of the
+[tree-sitter](https://tree-sitter.github.io/) parser generator and maintained by Tim Zakian.
+
+The plugin is platform-independent by utilizing a [WASM](https://webassembly.org/)-ified version of
+the Move parser included in this repository at
+[(./tree-sitter-move.wasm)](./tree-sitter-move.wasm). You can re-generate the WASM-ified version of
+the parser by running the [scripts/treesitter-wasm-gen.sh](scripts/treesitter-wasm-gen.sh) script
+(prerequisites for this are listed in the script itself). You should be careful when doing so, as
+certain changes to the parser may break the plugin (e.g., if parse tree node types are modified).
+
+## Prerequisites
+
+Currently, in order to use the plugin, you need to install `npm` command (`brew install npm` on a
+Mac). You can use the plugin to format Move files (`.move` extension) both on the command line and
+using Prettier's VSCode
+[extension](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode). When the
+plugin is complete, we will make it available directly from Move's VSCode extension.
+
+## Installation
+
+Currently, you need to install plugin from sources (when the plugin is complete, we will submit it
+to the NPM package [registry](https://www.npmjs.com/) for direct download).
+
+Clone Sui repository into `$SUI` directory:
+
+```bash
+git clone https://github.com/MystenLabs/sui.git "$SUI"
+```
+
+Go to `"$SUI"/external-crates/move/crates/move-analyzer/prettier-plugin` and run the following
```

Comment:
Is this still the correct path?

Reply:
README needs some special attention, I suggest trimming it here and opening a new PR with new guide / descriptions.